features/step_definitions/assertion_steps.rb

```diff
@@ -2,6 +2,16 @@
   expect(page).to have_content expected_text
 end
 
+Then("I should see the notice: {string}") do |expected_text|
+  notice = page.evaluate_script("document.querySelector('.notice');")
+  expect(notice.text).to eq expected_text
+end
+
+Then("I should see the alert: {string}") do |expected_text|
+  alert = evaluate_script("var notice = document.querySelector('.alert'); notice.innerText")
```

Comment:
This Will not work. Also, not used. You should have a sad path for the rating.

Reply:
We have a sad path user can only rate once or do you mean user is unable to rate?

Reply:
There’s an error handler that will kick in if a faulty  request comes through. You should have a scenario for that.